### PR TITLE
Update Android samples for non-optional messenger

### PR DESCRIPTION
**Pull request by tcobbs-bentley**
__NOTE:__ This PR is dependent on [this PR](https://github.com/iTwin/mobile-sdk-android/pull/22) in mobile-sdk-android.

### CODE REVIEW

File: Android/CameraSample/app/src/main/java/com/bentley/sample/camera/CameraITMApplication.kt
Comment (tcobbs-bentley) [inline Android/CameraSample/app/src/main/java/com/bentley/sample/camera/CameraITMApplication.kt:26]: Don't approve this PR until the mobile-sdk-android one has been merged.

Comment (tcobbs-bentley) [reply Android/CameraSample/app/src/main/java/com/bentley/sample/camera/CameraITMApplication.kt:26]: Other PR is approved. @toddsouthenbentley @simihartstein

**Review by toddsouthenbentley — APPROVED** [review]

**Pull request merged**
